www/content/docs/home/getting-started.mdx

```diff
@@ -56,3 +57,24 @@ const Artworks = props => {
   )
 }
 ```
+
+In order to use Artsy's fonts, you'll also want to add our webfont file to the
+CSS in the header of your app:
+
+```
+// Example using Styled Components:
+import { createGlobalStyle, css } from "styled-components"
+
+const GlobalStyles = createGlobalStyle<P>`
+  @import url("https://webfonts.artsy.net/all-webfonts.css");
+`
+
+// React Helmet example:
```

Comment:
Each example section can be broken out across the markdown so that more details / links can be added, e.g., 

```
### [Styled Components](https://www.styled-components.com/docs/api#createglobalstyle) example:
... add a code block ... 

### [React Head](..link to react head docs) example: 
... add a code block ...
```
and so on.